Test/dafny0/ByMethod.dfy.expect

```diff
@@ -1,17 +1,17 @@
 ByMethod.dfy(38,18): Error: this invariant could not be proved to be maintained by the loop
 ByMethod.dfy(38,18): Related message: loop invariant violation
 ByMethod.dfy(42,4): Error: a postcondition could not be proved on this return path
-ByMethod.dfy(35,7): Related location: this is the postcondition that could not be proved
+ByMethod.dfy(35,4): Related location: this is the postcondition that could not be proved
 ByMethod.dfy(47,11): Error: a postcondition could not be proved on this return path
 ByMethod.dfy(48,12): Related location: this is the postcondition that could not be proved
 ByMethod.dfy(55,11): Error: a postcondition could not be proved on this return path
 ByMethod.dfy(56,12): Related location: this is the postcondition that could not be proved
 ByMethod.dfy(60,4): Error: a postcondition could not be proved on this return path
-ByMethod.dfy(59,7): Related location: this is the postcondition that could not be proved
+ByMethod.dfy(59,4): Related location: this is the postcondition that could not be proved
```

Comment:
What is the rational of moving the highlight from the "method  to the "by", if any?